packages/react-core/src/components/Label/Label.tsx

```diff
@@ -27,8 +27,8 @@ export interface LabelProps extends React.HTMLProps<HTMLSpanElement> {
   onEditComplete?: (event: MouseEvent | KeyboardEvent, newText: string) => void;
   /** @beta Callback when an editable label cancels an edit. */
   onEditCancel?: (event: KeyboardEvent, previousText: string) => void;
-  /** Flag indicating the label text should be truncated. */
-  isTruncated?: boolean;
+  /** Css property expressed in any valid length unit that is used to modify the label text's max-width, after which any additional text will be truncated */
```

Comment:
Nit but not a blocker: Not sure if we necessarily need to mention that this is altering a CSS property:

```suggestion
  /** The max width of the label before it is truncated. */
```

Could also add, "Can be any valid CSS unit, such as '100%', '100px', or '16ch'."